accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx

```diff
@@ -704,7 +709,62 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
               </ol>
             </div>
 
-            <h3 id="a11y_assess">6.2 Accessibility Assessment</h3>
+            <h3 id="focus_view">6.2 Focus View</h3>
+            <div className="pa">
+              The <strong>'Focus view'</strong> switch allows you to to switch between viewing all issues on the page, or only the issues for a selected element or component in the DOM. To focus on any individual element or component:
```

Comment:
switch allows you to to switch. Just one "to"

Reply:
Done!